lib/src/widgets/auth_card.dart

```diff
@@ -518,6 +518,7 @@ class _LoginCardState extends State<_LoginCard> with TickerProviderStateMixin {
       loadingController: _loadingController,
       interval: _nameTextFieldLoadingAnimationInterval,
       labelText: messages.usernameHint,
+      autofillHints: [AutofillHints.email],
```

Comment:
shouldn't this be `AutofillHints.username`?

Reply:
Hi @everters, 

I think it is `[AutofillHints.email]` is the right choice for this part, as you can check at you next comment, it has `TextInputType.emailAddress`. It is indeed expected an email in this place, most auth services providers uses email to login, so it would be a breaking change to change this to `[AutofillHints.username]`. 

What can be done, if you think you would need username instead of email is to open a new PR to implement a new params like: `LoginType.username` or `LoginType.email` with default value `LoginType.email` to not break any existing implementation (legacy support)

What I agree is that `messages.usernameHint` is misleading and should be renamed to something like `messages.emailHint`.